### ISSUE TITLE
Replace the text "Execute" with "Initiate"

### ISSUE THREAD

**Issue by BartuCulha** [PrimeDAO/prime-deals-dapp#447]
We should replace instances of "Executed", "Execution", "Execute" with "Initiated", "Initiation", "Initiate" across the dapp.

![Create-a-Partnered-Deal-Prime-Deals (3).png](https://images.zenhubusercontent.com/609a9f5f0a40bdfde66fe338/460ac765-912b-4cbb-bcec-08896d9c0471)

@blink-km @dkent600 

Could you review this ticket please? I hope "Initiation" doesn't sound like a ritual

**Comment by BartuCulha** [comment]
If it makes sense, I can create a separate card for ""Execution period” will be replaced with “Funding period” @dkent600

**Comment by curvelabs-dev** [comment]
> If it makes sense, I can create a separate card for ""Execution period” will be replaced with “Funding period”

is ok as-is, thanks

**Comment by curvelabs-dev** [comment]
@BartuCulha 

> Could you review this ticket please? I hope "Initiation" doesn't sound like a ritual

ha ha good thought, but it would be hard to make that mistake.

**Comment by curvelabs-dev** [comment]
I don't think “Execution period” is synonymous with “Funding period”.  Once funding is complete, the deal is "executed" (contract lingo) which enabled swapping to begin.
